src/main/java/org/verdictdb/core/querying/SelectAllExecutionNode.java

```diff
@@ -77,6 +84,7 @@ public SqlConvertible createQuery(List<ExecutionInfoToken> tokens) throws Verdic
   @Override
   public ExecutionInfoToken createToken(DbmsQueryResult result) {
     ExecutionInfoToken token = new ExecutionInfoToken();
+    result.getMetaData().isAggregate = isAggregate;
```

Comment:
Is this always true?

Reply:
isAggregate field is a list of boolean of which size equals #column. It indicates which columns is aggregated, which will be used when estimating accuracy.